lib/newman_conway.js

```diff
@@ -1,10 +1,26 @@
 
-// Time Complexity:
-// Space Complexity:
+// Time Complexity: O(n)
+// Space Complexity: O(n)
 
 function newmanConway(num) {
-  throw new Error("Function not implemented yet...")
-}
+  if (num === 0) throw new Error();
```

Comment:
We should raise this error for _any_ value below the valid starting point of the sequence:
```js
  if (num < 1) throw new Error();
```